pkg/ccl/changefeedccl/changefeed_stmt.go

```diff
@@ -1362,6 +1401,40 @@ func (b *changefeedResumer) resumeWithRetries(
 	return errors.Wrap(ctx.Err(), `ran out of retries`)
 }
 
+func resolveDest(ctx context.Context, execCfg *sql.ExecutorConfig, sinkURI string) string {
+	u, err := url.Parse(sinkURI)
```

Comment:
i'm pretty sure i believe that returning "" here on parse failure will dtrt, but it's not super clear to me. maybe it would be clearer to return err from this function.

Reply:
Yeah, returning an error probably makes sense. I'll do that.